src/core/killall.c

```diff
@@ -205,10 +204,11 @@ static int killall(int sig, Set *pids, bool send_sighup) {
                 }
         }
 
-        return set_size(pids);
+        return nb_kill;
 }
 
-void broadcast_signal(int sig, bool wait_for_exit, bool send_sighup, usec_t timeout) {
+int broadcast_signal(int sig, bool wait_for_exit, bool send_sighup, usec_t timeout) {
+        int nb_child_left;
```

Comment:
and here too

Reply:
also, please rename nb_child_left → to "n_children_left" or so? i.e. plural sounds more appropriate here